indico/htdocs/js/indico/modules/reporter.js

```diff
@@ -47,5 +53,9 @@
 
     global.setupReporter = function() {
         setupStaticURLGeneration();
+
+        $('.report').on('indico:htmlUpdated', function() {
+            handleRowSelection();
+        }).trigger('indico:htmlUpdated');
```

Comment:
i'd rather call the function manually. htmlUpdated is always called with an argument, so calling it like this could break other code that also uses this event